packages/modelviewer.dev/styles/docs.css

```diff
@@ -190,3 +172,35 @@ hr.solid {
 .bolden {
   font-weight: bold;
 }
+
+/* Smaller screen/ mobile */
+@media only screen and (max-width: 800px) {
+  :root {
+    --sidebar-width: 0;
+  }
+
+  .sidebar {
+    width: var(--sidebar-width);
```

Comment:
The sticky header could use perhaps a bit of grey background so it doesn't blend so much into the white page.

Reply:
at the moment, the only way to do this and have it look good is with a subtle gradient, because the header is embedded inside the content div, the gradient doesn't look too bad, in the new PR, so I'll guess we'll see what your opinion on it is.